src/artemisa/component.js

```diff
@@ -26,23 +18,7 @@ class AbstractWithFetches extends React.Component {
 
   tryToFetch() {
     const { state, dispatch } = this.props;
-    this.getFetches().forEach(fetch => {
-      this.dispatchFetch(fetch, state, dispatch)
-    })
-  }
-
-  dispatchFetch({ storeFieldName, call, on }, state, dispatch) {
-    if (!on(this.props, state)) {
-      return;
-    }
-    const action = ({
-      type: `${ARTEMISA}_${storeFieldName}`,
-      dataApiCall: call(this.props, state)
-    })
-    const should = shouldFetch(state.artemisa[storeFieldName], action.dataApiCall.path, s => s.path)
-    if (should) {
-      dispatch(action)
-    }
+    dispatchFetches(this.props, state, dispatch, this.getFetches())
```

Comment:
Also probably out of the scope of this PR, but how do you feel about trying to reduce the amount of parameters we use in our functions by accepting an object or maybe a main parameter and an object?
I've seen this in some other parts of the app, and this may lead to force our users to send `undefined` parameters in order to use another one which is expected after.
To put an example, I mean replacing this with `dispatchFetches({props, state, dispatch, fetches})`.
What do you think?